server/methods/catalog.js

```diff
@@ -1493,7 +1493,7 @@ Meteor.methods({
     }).fetch();
     let variantValidator = true;
 
-    if (typeof product === "object" && product.title.length > 1) {
+    if (typeof product === "object" && product.title && product.title.length > 1) {
```

Comment:
previously, when the test which runs this code ran, we ensured that an Access Denied error was not being thrown. It wasn't (and was therefore passing) because `product.title.length` was throwing a `"Cannot read property 'length' of undefined"` error instead.